### PR TITLE
feat: add navigation and about page

### PR DESCRIPTION
**Pull request by braaar**
This PR includes #333 because my merging procedure was suboptimal. There are a few changes made to the about page after the fact (styling)

We must use `theme-ui`'s `Image`.

> Currently, it is not possible to take advantage of the Image component when creating a fully-static site with NextJS. Image optimization at build time is not supported out of the box. What is worse, there is [little mention](https://nextjs.org/docs/migrating/from-gatsby#migrating-from-gatsby-image) of this limitation in the NextJS docs, leading many developers, including myself, to discover it the hard way - by trying to use next/image with next export and discovering that it simply does not working.

[Source](https://blog.whereisthemouse.com/image-optimization-for-static-nextjs-sites)

### CODE REVIEW

File: next.config.js [hunk 1]
```diff
@@ -1,6 +1,9 @@
 /** @type {import('next').NextConfig} */
 const nextConfig = {
   reactStrictMode: true,
+  images: {
+    loader: 'custom',
```
Comment (cobraz) [inline next.config.js:5]: Do you need that? re cobraz/next-theme-ui#11

Comment (braaar) [reply next.config.js:5]: If I don't have this in the config I get this error: 

```
Error: Image Optimization using Next.js' default loader is not compatible with `next export`.
  Possible solutions:
    - Use `next start` to run a server, which includes the Image Optimization API.
    - Use any provider which supports Image Optimization (like Vercel).
    - Configure a third-party loader in `next.config.js`.
    - Use the `loader` prop for `next/image`.
  Read more: https://nextjs.org/docs/messages/export-image-api
    at /Users/brage/Kode/bjerk/website/node_modules/next/dist/export/index.js:257:23
    at async Span.traceAsyncFn (/Users/brage/Kode/bjerk/website/node_modules/next/dist/trace/trace.js:75:20)
error Command failed with exit code 1.
```

Comment (cobraz) [reply next.config.js:5]: Have you fixed `next-theme-ui` to version 1.2.0?

Comment (braaar) [reply next.config.js:5]: Just tried this. Downgraded to 1.2.0, deleted `node_modules`, `yarn.lock` and all build outputs. Still getting the same error. The next config thing seems necessary :/

Comment (cobraz) [reply next.config.js:5]: Okey, it's probably fine for now anyway. :)

**Review by cobraz — APPROVED** [review]

**Pull request merged**